app/views/backend/land_parcels/show.html.haml

```diff
@@ -6,9 +6,10 @@
   - c.cobble :general_informations do
     = product_mini_map
     = infos do
-      .med-info
-        %span.title= LandParcel.human_attribute_name(:population)
-        %span.value= resource.population.round(3).l + " " + resource.unit_name
+      - if resource.initial_shape
+        .med-info
+          %span.title= LandParcel.human_attribute_name(:net_surface_area)
+          %span.value= resource.initial_shape_area.round(2).in(:hectare)
```

Comment:
I understand the why but, we can't put `:hectare` everywhere like that, this won't work for small land parcels which are in square meters. I propose to use the `Activity#size_unit` to define which unit to use. A delegation should be added in `LandParcel` model or a method like `human_initial_shape_area` that will return good value with good unit.